tests/real/community.rs

```diff
@@ -61,6 +61,73 @@ async fn roundtrip() {
     assert!(drained);
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn server_state() {
+    skip_check!();
+
+    let local = "server_state";
+
+    // prepare a worker
+    let mut w = WorkerBuilder::default();
+    w.register(local, move |_| async move { Ok::<(), io::Error>(()) });
+    let mut w = w.connect(None).await.unwrap();
+
+    // prepare a producing client
+    let mut client = Client::connect(None).await.unwrap();
+
+    // examine server state before pushing anything
+    let server_state = client.info().await.unwrap();
+    assert!(server_state.faktory.queues.get(local).is_none());
+    // the following two assertions are not super-helpful but
+    // there is not much info we can make meaningful assetions on anyhow
+    // (like memusage, server description string, version, etc.)
+    assert!(server_state.server.connections >= 2); // at least two clients from the current test
+    assert!(server_state.server.uptime > 0); // if IPC is happenning, this should hold :)
+
+    // push 1 job
+    client
+        .enqueue(
+            JobBuilder::new(local)
+                .args(vec!["abc"])
+                .queue(local)
+                .build(),
+        )
+        .await
+        .unwrap();
+
+    // we only pushed 1 job on this queue
+    let server_state = client.info().await.unwrap();
+    assert_eq!(*server_state.faktory.queues.get(local).unwrap(), 1);
+    assert!(server_state.faktory.total_enqueued >= 1); // at least 1 job from this test
+    assert!(server_state.faktory.total_queues >= 1); // at least 1 qeueu from this test
+
+    // let's know consume that job ...
+    assert!(w.run_one(0, &[local]).await.unwrap());
+
+    // ... and verify the queue has got 0 pending jobs
+    //
+    // NB! If this is not passing locally, make sure to launch a fresh Faktory container,
+    // because if you have not pruned its volume the Faktory will still keep the queue name
+    // as registered.
+    // But generally, we are performing a clean-up by consuming the jobs from the local queue/
+    // and then deleting the queue programmatically, so there is normally no need to prune docker
+    // volumes to perform the next test run. Also note that on CI we are always starting a-fresh.
+    let server_state = client.info().await.unwrap();
+    assert_eq!(*server_state.faktory.queues.get(local).unwrap(), 0);
+    assert!(server_state.faktory.total_processed >= 1); // at least 1 job from this test
+
+    client.queue_remove(&[local]).await.unwrap();
```

Comment:
Now that we got this command, this can be utilized for clean ups

While not crucial on CI, can make life easier when developing locally: there will be no remainders from the previous test runs.